test.html

```diff
@@ -21,11 +21,13 @@
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
```

Comment:
Could we move test.html and test-legacy.html to `test/` maybe? Or `test/index.html` and `test/legacy.html`? Could also contain a README about writing tests (in the future) -- main rational being to minimize things in the root

Reply:
👍

Reply:
Done